public/scripts/extensions/regex/style.css

```diff
@@ -1,5 +1,9 @@
 @import "debugger.css";
 
+.regex_settings [hidden] {
+    display: none !important;
```

Comment:
Does this also work without `!important`? Maybe with a more specific selector?
Adding this is always risky, especially for anything that CSS styles might want to do.